### ISSUE TITLE
error reporting

### ISSUE THREAD

**Issue by NicoLeOca** [pymedusa/Medusa#1329]
### Before submitting your issue:

Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: last dev
OS: windows 10

2016-10-26 10:24:03 ERROR    Thread-31 :: [9f9a897] Failed doing web ui callback: Traceback (most recent call last):
  File "C:\Medusa\medusa\server\web\core\base.py", line 271, in async_call
    result = function(**kwargs)
  File "C:\Medusa\medusa\server\web\home\post_process.py", line 49, in processEpisode
    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=type, ignore_subs=argToBool(ignore_subs)
  File "C:\Medusa\medusa\processTV.py", line 287, in processDir
    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, ignore_subs, result)
  File "C:\Medusa\medusa\processTV.py", line 548, in process_media
    if has_matching_unknown_subtitles(cur_video_file_path):
  File "C:\Medusa\medusa\processTV.py", line 672, in has_matching_unknown_subtitles
    Language('und') in get_embedded_subtitles(video_path)
  File "C:\Medusa\medusa\processTV.py", line 683, in get_embedded_subtitles
    subliminal_video = scan_video(video_path)
  File "C:\Medusa\lib\subliminal\core.py", line 380, in scan_video
    video = Video.fromguess(path, guessit(path))
  File "C:\Medusa\lib\subliminal\video.py", line 93, in fromguess
    return Episode.fromguess(name, guess)
  File "C:\Medusa\lib\subliminal\video.py", line 166, in fromguess
    raise ValueError('Insufficient data to process the guess')
ValueError: Insufficient data to process the guess
Traceback (most recent call last):
  File "C:\Medusa\medusa\server\web\core\base.py", line 271, in async_call
    result = function(**kwargs)
  File "C:\Medusa\medusa\server\web\home\post_process.py", line 49, in processEpisode
    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=type, ignore_subs=argToBool(ignore_subs)
  File "C:\Medusa\medusa\processTV.py", line 287, in processDir
    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, ignore_subs, result)
  File "C:\Medusa\medusa\processTV.py", line 548, in process_media
    if has_matching_unknown_subtitles(cur_video_file_path):
  File "C:\Medusa\medusa\processTV.py", line 672, in has_matching_unknown_subtitles
    Language('und') in get_embedded_subtitles(video_path)
  File "C:\Medusa\medusa\processTV.py", line 683, in get_embedded_subtitles
    subliminal_video = scan_video(video_path)
  File "C:\Medusa\lib\subliminal\core.py", line 380, in scan_video
    video = Video.fromguess(path, guessit(path))
  File "C:\Medusa\lib\subliminal\video.py", line 93, in fromguess
    return Episode.fromguess(name, guess)
  File "C:\Medusa\lib\subliminal\video.py", line 166, in fromguess
    raise ValueError('Insufficient data to process the guess')
ValueError: Insufficient data to process the guess

full logs:
https://gist.github.com/NicoLeOca/4ec37df98fcce01d3255ba122fc6e46d

**Comment by medariox** [comment]
What's the state here? Fixed? @fernandog @ratoaq2

**Comment by ratoaq2** [comment]
Just starting to check it:

This is the important part of the log:
```
2016-10-26 10:24:03 DEBUG    Thread-31 :: [9f9a897] Parsed 'Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7' into title: Black Mirror, season: 2, screen_size: 1080p, format: WEB-DL, audio_codec: AAC, audio_channels: 2.0, video_codec: h264, release_group: Coo7, type: episode, absolute_episode: [], quality: 1080p WEB-DL, episode: []
```

What should be done when video has no episode, only season?

**Comment by duramato** [comment]
@ratoaq2 that is the directory.. it's of a season pack
@NicoLeOca what is the file list of `Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7`

**Comment by NicoLeOca** [comment]
@duramato 
Black.Mirror.S02E01.Be.Right.Back.1080p.WEB-DL.AAC2.0.H.264-Coo7
Black.Mirror.S02E02.White.Bear.1080p.Web-Dl.Aac2.0.h264-COO7
Black.Mirror.S02E03.The.Waldo.Moment.1080p.Web-Dl.Aac2.0.h264-COO7
Black.Mirror.S02.Special.White.Chirstmas.1080p.WEB-DL.AAC2.0.H.264-Coo7

**Comment by ratoaq2** [comment]
It's related to process_tv.py logic: has_matching_unknown_subtitles:

```
  File "C:\Medusa\medusa\server\web\home\post_process.py", line 49, in processEpisode
    is_priority=argToBool(is_priority), delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=type, ignore_subs=argToBool(ignore_subs)
  File "C:\Medusa\medusa\processTV.py", line 287, in processDir
    process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, ignore_subs, result)
  File "C:\Medusa\medusa\processTV.py", line 548, in process_media
    if has_matching_unknown_subtitles(cur_video_file_path):
  File "C:\Medusa\medusa\processTV.py", line 672, in has_matching_unknown_subtitles
    Language('und') in get_embedded_subtitles(video_path)
  File "C:\Medusa\medusa\processTV.py", line 683, in get_embedded_subtitles
    subliminal_video = scan_video(video_path)
  File "C:\Medusa\lib\subliminal\core.py", line 380, in scan_video
    video = Video.fromguess(path, guessit(path))
  File "C:\Medusa\lib\subliminal\video.py", line 93, in fromguess
    return Episode.fromguess(name, guess)
  File "C:\Medusa\lib\subliminal\video.py", line 166, in fromguess
    raise ValueError('Insufficient data to process the guess')
ValueError: Insufficient data to process the guess
```

It seems that's trying to process the folder as a video. @fernandog

**Comment by fernandog** [comment]
the problem is the special episode.

there is no episode so this fails `return Episode.fromguess(name, guess)`

that's why i tried to ignore special episodes parsing with this PR:
Raise InvalidNameException for special episodes
https://github.com/pymedusa/Medusa/pull/1317

i see no other solution

**Comment by duramato** [comment]
The issue is in subliminal tho

**Comment by fernandog** [comment]
What report guessit say passing folder + special? On mobile now

**Comment by fernandog** [comment]
no idea why it fails @ratoaq2 

## report_guessit:
```
python report_guessit.py "/media/series/Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7\Black.Mirror.S02.Special.White.Chirstmas.1080p.WEB-DL.AAC2.0.H.264-Coo7.mkv"
# guessit: 2.1.1.dev0  rebulk: 0.7.6
? /media/series/Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7\Black.Mirror.S02.Special.White.Chirstmas.1080p.WEB-DL.AAC2.0.H.264-Coo7.mkv
: season: 2
  screen_size: 1080p
  format: WEB-DL
  audio_codec: AAC
  audio_channels: 2.0
  video_codec: h264
  release_group: Coo7
  title: Black Mirror
  episode_title: Special White Chirstmas
  episode_details: Special
  container: mkv
  mimetype: video/x-matroska
  type: episode

```

## guessit
```
guessit "/media/series/Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7\Black.Mirror.S02.Special.White.Chirstmas.1080p.WEB-DL.AAC2.0.H.264-Coo7.mkv"
For: /media/series/Black.Mirror.S02.1080p.WEB-DL.AAC2.0.H.264-Coo7\Black.Mirror.S02.Special.White.Chirstmas.1080p.WEB-DL.AAC2.0.H.264-Coo7.mkv
GuessIt found: {
    "season": 2,
    "screen_size": "1080p",
    "format": "WEB-DL",
    "audio_codec": "AAC",
    "audio_channels": "2.0",
    "video_codec": "h264",
    "release_group": "Coo7",
    "title": "Black Mirror",
    "episode_title": "Special White Chirstmas",
    "episode_details": "Special",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode"
}

```

**Comment by ratoaq2** [comment]
It should be fixed by #1452
